File: command_help.go
package main

import (
	"fmt"
)

func commandHelp(config *Config) {
	helpMessage := "Welcome to the Pokedex!\nUsage:\n\n"
	fmt.Print(helpMessage)

	for _, cmd := range getCommands() {
		fmt.Printf("%s: %s\n", cmd.name, cmd.description)
	}
	fmt.Println()
}

File: src/commands/command_help.ts
import { State } from "../state.js";

export async function commandHelp(state: State): Promise<void> {
    console.log("Welcome to the Pokedex!");
    console.log("Usage:\n");

    for (const key in state.cmdRegistry){
        const command = state.cmdRegistry[key];

        console.log(`${command.name}: ${command.description}`);
    }
    return;
};